src/Illuminate/Support/Arr.php

```diff
@@ -120,7 +120,7 @@ public static function exists($array, $key)
             return $array->offsetExists($key);
         }
 
-        return array_key_exists($key, $array);
+        return isset($array[$key]) || array_key_exists($key, $array);
```

Comment:
I only test this on local machine, and adding isset is faster.

However, when using sandbox online, array_key_exists is faster.

Benchmark : http://sandbox.onlinephpfunctions.com/code/90069414dc1b46c666fbfd96e1b9415b9d151378